proto/regen/ecocredit/v1/tx.proto

```diff
@@ -23,6 +24,17 @@ service Msg {
   // be distributed to recipients in either tradable or retired form.
   rpc CreateBatch(MsgCreateBatch) returns (MsgCreateBatchResponse);
 
+  // MintBatchCredits issues new token in a given batch.
+  // The issuer must be the account who created (or delegated using x/authz),
+  // the batch.
+  // The request will fail if the batch is not open.
+  // NOTE: this method is only for bridge purpose. It must not be used
+  // for issuing native credits on Regen. More specifically, we
+  // enable minting more credits in an existing batch, when the batch
+  // represents a vintage originally registered in another chain.
+  rpc MintBatchCredits(MsgMintBatchCredits)
+      returns (MsgMintBatchCreditsResponse);
```

Comment:
Can you comment this line out for the time being and then we can uncomment when implementing (#973). Otherwise anyone who makes proto changes will face errors because this message is not yet implemented.

Reply:
```suggestion
  // rpc MintBatchCredits(MsgMintBatchCredits)
      returns (MsgMintBatchCreditsResponse);
```